src/components/Form.js

```diff
@@ -74,42 +105,46 @@ export default class Form extends Component {
     return null;
   }
 
-  onChange = (formData, options={validate: false}) => {
+  onChange = (formData, options = {validate: false}) => {
```

Comment:
Definitely! Please add a rule, I think in this project we don't use spaces around `=` for default arg values. I don't have super strong opinion on this but the important thing is consistency.